vagrant/README.md

```diff
@@ -0,0 +1,35 @@
+# Vagrant
+
+Please install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/). After that you can run the Vagrant environment with just one command:
```

Comment:
Nitpick: I'd maybe introduce a guideline for writing markdown files so that we wrap around (loosely enforced) 80-100 characters and/or full sentences.

This way it will be more readable for the person editing the markdown file and while being visualized it will render it the same way (without a new line - as long as there is no empty line separating the sentences).

i.e. this could become:

```suggestion
Please install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/).
After that you can run the Vagrant environment with just one command:
```